WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -291,7 +286,7 @@ private void newPost() {
         if (!isAdded()) {
             return;
         }
-        ActivityLauncher.addNewPostOrPageForResult(getActivity(), mSite, mIsPage, false);
+        ActivityLauncher.addNewPostOrPageForResult(getActivity(), mSite, false, false);
```

Comment:
This method is never called with `isPage = true`, so it can be modified only for posts.

Reply:
Very nice catch, I didn't think to check this at all. Fixed in 78db0202950f3a865e8cb3f63872f9495134d43f.